django/thunderstore/community/models/community.py

```diff
@@ -104,6 +104,9 @@ class Community(TimestampMixin, models.Model):
         default=OptionalBoolChoice.NONE,
     )
 
+    # Will hide/show "Install with Mod Manager" button on package pages
+    mod_manager_support = models.BooleanField(default=True)
```

Comment:
We've used a naming convention where boolean fields are generally prefixed with `is_` or `has_` etc to make it more clear, could do `has_mod_manager_support` here for example.